base/src/schema.ts

```diff
@@ -914,5 +912,5 @@ function findRelations(
   return find()
 }
 
-export { Schema, ValidationError, findRelations }
-export type { SchemaDefinitions, Definition, DefinitionType }
+export { Schema, ValidationError, findRelations, getSchemaDefinition }
```

Comment:
You didn't have to export `getSchemaDefinition`. It's private to `schema.ts`

To test `getSchemaDefinition` you need to use a schema that looked like this:

```ts
const schema = {
  address: { type: 'object', schema: 'address' }
}
```

This will make the code branch to the `getSchemaDefinition` part of the code.

To elaborate, when you pass a string as the schema for an object, it's known as a schema ref.